tests/mvp-demo/mvp-demo.go

```diff
@@ -7,8 +7,8 @@ import (
 	"time"
 
 	"github.com/devfile/library/pkg/util"
+	ecp "github.com/enterprise-contract/enterprise-contract-controller/api/v1alpha1"
```

Comment:
I think you need also to update on `tests/build/build_templates.go`

Reply:
That was it! Thank you very much :pray: